app/src/main/java/com/bucketlist/dao/ILocalBucketListItemDAO.kt

```diff
@@ -6,15 +6,15 @@ import com.bucketlist.dto.BucketListItem
 
 @Dao
 interface ILocalBucketListItemDAO {
-    @Query("SELECT * FROM bucketlistitem")
+    @Query("SELECT * FROM bucketListItem")
     fun getAll()  : LiveData<List<BucketListItem>>
 
     @Insert(onConflict = OnConflictStrategy.REPLACE)
-    fun insertAll(bucketlistitems: BucketListItem)
+    fun insertAll(bucketListItems: BucketListItem)
 
     @Delete
-    fun delete(bucketlistitem: BucketListItem)
+    fun delete(bucketListItem: BucketListItem)
 
     @Insert
-    fun save(bucketlistitem: BucketListItem)
+    fun save(bucketListItem: BucketListItem)
```

Comment:
It is general and best practice in coding and programming to use camelCase. This is the most basic rule of coding-
conventions and should be followed as mentioned here: https://whatis.techtarget.com/definition/CamelCase